### PR TITLE
Update tidy eval template

### LINKED ISSUE #1008: install covr  when using `use_github_actions_tidy()`
Unless `covr` is added to a package's DESCRIPTION as a suggest, you get: 

image

after setting actions with `usethis::use_github_actions_tidy()`

It's easy enough to add `covr` to the suggests but perhaps it could be done as part of basic setup ? or maybe this is an issue of `usethis::` ?

### PR DESCRIPTION
**Pull request by lionel-**
Fixes r-lib/rlang#1008.

- Remove more advanced helpers.
- Mention modern syntax.
- Add examples.

### CODE REVIEW

**Comment by malcolmbarrett** [comment]
A trick that I've used to get around that is to treat `{{` and `}}` like data, e.g. use `{{{ curly_open }}}` in the template, then `data = list(curly_open = "{{")` in `use_template()`.

File: inst/templates/tidy-eval.R [hunk 1]
```diff
@@ -1,47 +1,107 @@
...
+#'       summarise(mean = mean({{ var }}))
+#'   }
+#'   ```
+#'
+#' * [enquo()] and [enquos()] delay the execution of one or several
+#'   function arguments. The former returns a single expression, the
+#'   latter returns a list of expressions. Once defused, expressions
+#'   will no longer evaluate on their own. They must be injected back
+#'   in evaluation context with `!!` (for single expressions) and
+#'   `!!!` (for lists of expressions).
```
Comment (jennybc) [inline inst/templates/tidy-eval.R:37]: Something seems off with this wording. Not 100% my proposal is right. I also think singular is more clear here.
```suggestion
#'   into an evaluation context with `!!` (for a single expression) and
#'   `!!!` (for a list of expressions).
```

File: inst/templates/tidy-eval.R [hunk 2]
```diff
@@ -1,47 +1,107 @@
...
+#'     # Use `{{` to tunnel function arguments and the usual glue
+#'     # operator `{` to interpolate plain strings.
+#'     data %>%
+#'       summarise("{{ var }}_mean_{suffix}" := mean({{ var }}))
+#'   }
+#'   ```
+#'
+#' * Many tidy eval functions like `dplyr::mutate()` or
+#'   `dplyr::summarise()` give an automatic name to unnamed inputs. If
+#'   you need to create the same sort of autonamic names by yourself,
```
Comment (jennybc) [inline inst/templates/tidy-eval.R:82]:
```suggestion
#'   you need to create the same sort of automatic names by yourself,
```

**Review by jennybc — COMMENTED** [review]
I'll think about how to handle the clash between the content of this template and whisker. This template does not actually *use* whisker, so it's sort of sad that they are in tension.

Since the new template uses markdown syntax, I think `use_tidy_eval()` also needs to gain some logic to make sure the host package is using markdown with roxygen2.

**Comment by jennybc** [comment]
> IIUC this is taken care of by the @md tag which enables markdown in single topic pages.

But I think the host package still needs to have `Roxygen: list(markdown = TRUE)` in DESCRIPTION, yeah?

**Comment by jennybc** [comment]
Based on a small experiment, my concern about needing `Roxygen: list(markdown = TRUE)` doesn't seem to be warranted.

**Pull request merged**